eva-accession-core/pom.xml

```diff
@@ -25,6 +29,13 @@
             <groupId>uk.ac.ebi.ampt2d</groupId>
             <artifactId>accessioning-commons</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.fakemongo</groupId>
+            <artifactId>fongo</artifactId>
+            <scope>test</scope>
+            <version>${com.github.fakemongo.version}</version>
```

Comment:
You don't need to do this: specify the dependency in the parent like [this](https://github.com/EBIvariation/eva-accession/blob/develop/pom.xml#L29-L37) and it will only be inherited by those modules that specify it. I verified it yesterday after we discussed it at the office, using `mvn dependency:tree`.